bin/ch-validate

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+. $(dirname "$0")/base.sh
+
+usage () {
+   cat 1>&2 <<EOF
+Verify that a specified Charliecloud image has the basic required structure to run.
+
+Usage:
+
+   $ $(basename $0) IMAGE_DIR
+
+Arguments:
+   
+   IMAGE_DIR   Directory containing the Charliecloud image
+
+EOF
+   exit ${1:-1}
+}
+
+need_files=( 
+   "/etc/passwd"
+   "/etc/group"
+   "/etc/hosts"
+   "/etc/resolv.conf"
+)
+
+need_dirs=(
+   "/etc"
+   "/dev"
+   "/proc"
+   "/sys"
+   "/tmp"
+)
+
+
+if [ "$1" = "--help" ]; then
+    usage 0
+fi
+if [ "$1" = "--version" ]; then
+    version
+    exit 0
+fi
+
+if [ $# -ne 1 ]; then
+   usage -1
+fi
+
+image_dir=$1
+
+if [ ! -d $image_dir ]; then
+   echo "Image Directory does not exist: $image_dir is not a directory."
```

Comment:
Perhaps simply `"$image_dir is not a directory"`?

Reply:
Seems like the error still applies?  If it happens to be a file, there's still not an image directory there.